presto-main/src/test/java/com/facebook/presto/operator/scalar/TestFormatUtilsFunctions.java

```diff
@@ -0,0 +1,69 @@
+/*
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package com.facebook.presto.operator.scalar;
+
+import org.testng.annotations.Test;
+
+import static com.facebook.presto.spi.type.VarcharType.VARCHAR;
+
+public class TestFormatUtilsFunctions
+        extends AbstractTestFunctions
+{
+    @Test
+    public void testFormatCount()
```

Comment:
Please test rounding explicitly showing intentions (this is good candidate for separate test function).
Include some cases of rounding up.
Note that there is explicit assumption about rounding mode in the code (`format.setRoundingMode(RoundingMode.HALF_UP);`), so it's nice to express that in tests as well.